en/editor/project/index.md

```diff
@@ -169,72 +169,7 @@ Example:
-
-- `gravity` Gravity direction vector, the sign means the positive or negative direction on the axis. **Default:** `{ x: 0, y: -10, z: 0 }`.
-- `allowSleep` Whether to allow rigid bodies to enter sleep state. **Default:** `true`.
-- `sleepThreshold` The maximum speed threshold for entering sleep. **Default:** `0.1`，**Min:** `0`.
-- `autoSimulation` Whether to enable automatic simulation.
-- `fixedTimeStep` Fixed time step between each simulation. **Default:** `1/60`, **Min:** `0`.
-- `maxSubSteps` Maximum number of substeps per simulation step. **Default:** `1`, **Min:** `0`.
-- `friction` Coefficient of friction. **Default:** `0.5`.
-- `rollingFriction` Rolling friction coefficient. **Default:** `0.1`.
-- `spinningFriction` Spin friction coefficient. **Default:** `0.1`.
-- `restitution` Coefficient of elasticity. **Default:** `0.1`.
-- `useCollisionMatrix` Whether to use collision matrix. **Default:** `true`.
-- `collisionMatrix` The setting result of the collision matrix. **Format:** `{ index: value }`, **Default:** `{ "0": 1 }`.
-<!-- - `useNodeChains` Whether to use a node chain to combine rigid bodies. **Default:** `true`. -->
-
-### Collision Matrix
-
-The collision matrix is used to manage the group and mask of physical elements. The corresponding mask value will be set automatically after `useCollisionMatrix` is turned on.This feature is still in its infancy, so please use it with caution.
-
-![Physics-collision](./index/physics-collision.png)
-
-#### Grouping concept
-
-In the editor, the grouping format of the collision matrix is `{index, name}`, `index` is the number of bits from `0` to `31`, and `name` is the name of the group. The new project project will have a default grouping: `{index: 0, name:'DEFAULT'}`.
-
-By clicking the `+` button you can add a new group.
-
-> **Note**: Both `index` and `name` cannot be empty and cannot be repeated with existing items; after adding, the group cannot be deleted, only the name of the group can be modified.
-
-![Physics-collision-add](./index/physics-collision-add.png)
-
-#### How to configure
-
-Take a new `water` group as an example:
-
-![Physics-collision-demo](./index/physics-collision-demo.png)
-
-This table lists all the groups, and you can check it to determine which two groups will do the collision detection.
-
-**As shown in the figure above, whether `DEFAULT` and `water` will perform collision detection will be determined by whether the corresponding check box is checked**.
-
-According to the above rules, the collision pairs generated in this table are:
-
-- DEFAULT - water
-- DEFAULT - DEFAULT
-
-And the grouping pairs that do not perform collision detection include:
-
-- water - water
-
-In addition, the `Group` property on the rigid body component needs to be configured into the corresponding physical element:
-
-![rigidbody-group](./index/rigidbody-group.jpg)
+Used to configure the parameters of the physics，For details, please refer to the [Physics Configs](physics-configs.md);
```

Comment:
Used to configure the parameters of the physics，For details, please refer to the [Physics Configs](physics-configs.md);
-->
Used to configure the parameters of the physics environment. For details, please refer to the [Physics Configs](physics-configs.md) documentation.